Code/GraphMol/Descriptors/Data3Ddescriptors.cpp

```diff
@@ -0,0 +1,42 @@
+#include <cstdlib>
+#include <iostream>
+#include "Data3Ddescriptors.h"
+
+using namespace std;
+
+
+double Data3Ddescriptors::mw[53]={0.084,0,0,0,0.900,1.000,1.166,1.332,1.582,0,0,0, 2.246, 2.339, 2.579,2.670, 2.952,0,0,0,0,0,0,0,0, 4.650, 4.907, 4.887, 5.291, 5.445,0,0,0,0, 6.653,0,0,0,0,0,0,0,0,0,0,0,0,0,0, 9.884,0,0, 10.56};
```

Comment:
A general comment on these arrays: they are all defined to be a fixed size, but in the rest of the code I don't think you actually check to make sure that the indices into them are valid. So, for example, if I have a molecule with Th in it, I will end up with a random value if I ask for the vdW value of it.
That bounds checking is needed somewhere...